src/Altinn.AccessManagement.Integration/Clients/ResourceRegistryClient.cs

```diff
@@ -98,5 +98,52 @@ public async Task<List<ServiceResource>> GetResources(List<string> resourceIds)
 
             return resources;
         }
+
+        /// <summary>
+        /// Get resource information for the the given list of resourceids
+        /// </summary>
+        /// <param name="scopes"> the scope of resources</param>
+        /// <returns></returns>
+        public async Task<List<ServiceResource>> SearchResources(string scopes)
+        {
+            List<ServiceResource> resources = new List<ServiceResource>();
+            List<ServiceResource> filteredResources = new List<ServiceResource>();
+            ResourceSearch resourceSearch = new ResourceSearch();
+            resourceSearch.ResourceType = ResourceType.MaskinportenSchema;
+
+            try
+            {
+                string endpointUrl = $"resourceregistry/api/v1/resource/search?search.resourcetype={ResourceType.MaskinportenSchema}";
```

Comment:
The lookup of resources based on ResourceType will be needed by others aswell so might as well put this in it's own function imidiately, which is also a very cache-able function.

In my view the Client should also just be a wrapper of the actual client/api call and not do additional filtering, which should instead be up to a service.